eras/allegra/impl/testlib/Test/Cardano/Ledger/Allegra/CDDL.hs

```diff
@@ -1,17 +1,22 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
 {-# LANGUAGE OverloadedLists #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE QuasiQuotes #-}
-{-# LANGUAGE NoImplicitPrelude #-}
-{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
-
 {-# HLINT ignore "Use camelCase" #-}
 {-# HLINT ignore "Evaluate" #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
```

Comment:
I would suggest removing this option. In case you need to use it for HLINT you an just use
```
{- HLINT blah -}
```
instead of making it a pragma. Same with Fourmolu